README.md

```diff
@@ -55,14 +64,39 @@ mkdir -p WORK_DIR/EXAMPLE_OUTPUT/
 mkdir -p WORK_DIR/EXAMPLE_PROJECT/
 ```
 
-4. Make a file with a list of images to run. For example, let's say we want to process all the `.tif` images in the source folder. We'll get the list of files, and put them into a text file in the project folder. This will be used by the workflow.
+You also need to put the `parse_csv.py` script in there, into a `templates` folder:
 
 ```bash
-cd WORK_DIR/EXAMPLE_PROJECT/
-ls WORK_DIR/EXAMPLE_IMAGES/*tif > filelist.txt
+mkdir -p WORK_DIR/EXAMPLE_PROJECT/templates/
+```
+
+Copy the `parse_csv.py` file so it is located at `WORK_DIR/EXAMPLE_PROJECT/templates/parse_csv.py`. You *must* do this, or else the script will not run.
+
+
+4. Make a CSV file with the list of images to run. This is created in CellProfiler.
+
+
+5. Make sure the file paths in the CSV will end up working
+
+If you're not running CellProfiler in the same way that you will run the workflow (e.g. 'On the compute cluster'), then you may need to use the `FILE_PREFIX_IN` and `FILE_PREFIX_OUT` fields.
```

Comment:
I would phrase this, "If you're not running this workflow on the same computer that you used to configure CellProfiler, it's possible that the file paths need to be modified slightly. Both systems may be accessing the same filesystem, but that filesystem may be mounted at a different base location. By using the `FILE_PREFIX_IN` and `FILE_PREFIX_OUT` fields, we can modify the image filepaths generated by CellProfiler in the CSV so that it matches the location which can be used to locate the same file on the computer being used to run the workflow."